app/views/main/ServiceOffScreens/ExposureNotificationNotAvailable.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useAssets } from '../../../TracingStrategyAssets';
+import { ServiceOffScreen } from './Base';
+
+export const ExposureNotificationNotAvailableScreen = (): JSX.Element => {
+  const {
+    exposureNotificationsNotAvailableHeader,
+    exposureNotificationsNotAvailableSubheader,
+  } = useAssets();
+
+  return (
+    <ServiceOffScreen
```

Comment:
All of these various flavors of a `ServiceOffScreen` which are only different by a some conditionally rendered content suggests to me that we really have a domain concept of one screen that has a few different internal states rather than six truely separate screens.

While your code is very readable and makes sense, for long term maintainability its generally preferable to prefer fewer components/files where possible as this reduces the surface area of things that need to be understood and maintained. Another way i like to think about it is, we should be preferring internally managed stated over passed in props when possible, which i believe is encouraged pretty heavily in the react docs as well.

What do you think about having one `HomeIndicatorScreen` which understands how to render the correct content based off the state of the application?  something like:

```typescript
const HomeIndicatorScreen = ({ navigation }: HomeIndicatorScreenProps) => {
    const { permissionState } = useContext(myPermissionContext)
    const [myOtherPermissionState, setMyOtherPermissionState] = useState<MyPermissionState>({ ... })
    .... other bits of application state that matter

   const determineScreenContent = () => {
       if (!trackingInfo.canTrack) {
           return [tracingOffHeaderText, tracingOffSubheaderText]
        } else if (other possible state) {
          return ...
   }
    const [headerText, subheaderText] = determineScreenContent()

   return (
      <View>
            <Text>{headerText}</Text>
        ....
```


you could even get fancy and use a typescript union or enum to ensure that impossible states remain impossible with something like:

```typescript
type HomeIndicatorState = TracingEnabled | TracingOff | NotificationsOff | NoAuthorities | ... | Unknown


const HomeIndicatorScreen = ({ navigation }: HomeIndicatorScreenProps) => {
   const [indicatorState, setIndicatorState] = useState<HomeIndicatorState>(Unknown)

   const determineIndicatorState = async (...maybeSomeArgs): HomeIndicatorState => {
           const state = await ....logic
           ...do other logic
           setIndicatorState(theIndicatorState)
        }
        
     useEffect(() => {
           determineIndicatorState()
        ...
        },[])

       const inidicatorContent = () => {
             switch (indicatorState) {
                  case (TracingEnabled): {
                       return [enabledHeader, enabledSubHeader]
                  }
                  case (TracingOff): {
                       return [offHeader, offSubHeader]
            ...
        }

      const [headerText, subHeaderText] = indicatorContent()

         return (
               <View>
                     <Text>{headerText}</Text>
           ...
```

Reply:
Lengthy but a great piece of feedback that is much appreciated! That all makes a lot of sense to me, and I can see the benefits of keeping all of that logic consolidated in a single logical view.